viewer/packages/utilities/src/three/index.ts

```diff
@@ -1,8 +0,0 @@
-/*!
```

Comment:
Why was this file removed?

Reply:
It's not used anywhere, apparently.. Would there be a point in keeping it?

Reply:
Ok